### PR TITLE
Refactoting

### PR DESCRIPTION
**Pull request by Elena-Shagdurova**

### CODE REVIEW

**Review by agadjan — APPROVED** [review]

File: src/test/java/school/redrover/GroupUnicornsTest.java [hunk 1]
```diff
@@ -267,11 +267,10 @@ public void verificationSocialIconsGitHub2() throws InterruptedException {
             driver.quit();
         }
     }
-    @Ignore
     @Test
     public void unsuccessfulLoginDigitalBank() {
-        WebDriver driver = new ChromeDriver();
-        try {
+        WebDriver driver = getDriver();
```
Comment (PeterKuzyk) [inline src/test/java/school/redrover/GroupUnicornsTest.java:272]: Please remove this and use the method that Sergey created.

File: src/test/java/school/redrover/GroupUnicornsTest.java [hunk 2]
```diff
@@ -284,17 +283,13 @@ public void unsuccessfulLoginDigitalBank() {
...
-        } finally {
-            driver.quit();
-        }
     }
-    @Ignore
     @Test
     public void successfulLoginDigitalBank() {
-        WebDriver driver = new ChromeDriver();
-        try {
+        WebDriver driver = getDriver();
```
Comment (PeterKuzyk) [inline src/test/java/school/redrover/GroupUnicornsTest.java:292]: the same here

**Review by PeterKuzyk — CHANGES_REQUESTED** [review]

File: src/test/java/school/redrover/GroupUnicornsTest.java (hunk 1 above)
Comment (Elena-Shagdurova) [reply src/test/java/school/redrover/GroupUnicornsTest.java:272]: OMG I changed already. Why it looks like this? I'll do it. Thank you

**Comment by Elena-Shagdurova** [comment]
wrong code

**Pull request closed without merging**